ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceUserFixture.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.model.headers.Authorization
+import akka.stream.Materializer
+import com.daml.grpc.adapter.{AkkaExecutionSequencerPool, ExecutionSequencerFactory}
+import com.daml.http.HttpServiceTestFixture.{authorizationHeader, postRequest}
+import com.daml.http.util.ClientUtil.uniqueId
+import com.daml.jwt.JwtSigner
+import com.daml.jwt.domain.{Jwt, DecodedJwt}
+import com.daml.platform.sandbox.SandboxRequiringAuthorizationFuns
+import scalaz.syntax.show._
+import scalaz.syntax.tag._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait HttpServiceUserFixture {
+  protected def testId: String
+
+  // TODO(SC) we make asys, mat, and aesf "win" to match the prior behavior when
+  // these defns were in AbstractHttpServiceIntegrationTestFuns.  But it
+  // might be possible to phase them out entirely in favor of AkkaBeforeAndAfterAll
+  import shapeless.tag, tag.@@ // used for subtyping to make `AHS ec` beat executionContext
+  implicit val `AHS asys`: ActorSystem @@ this.type = tag[this.type] {
```

Comment:
The before and after all would take care of termination, now I can't see (immediately) where this is done, probably somewhere else in the code. (just started review)

Reply:
Maybe this does not change, so just to verify, can you point to where the actorsystem gets terminated? I think it makes sense to do the BeforeAndAfterAll, inthis PR or in a follow up.

Reply:
Can you elaborate further here? I need to make sure that I understand how tagging changes priorities for the implicit resolution

Reply:
The system has never been terminated; as far as I know it this ActorSystem definition even predates `AkkaBeforeAndAfterAll`, it certainly predates the `daml` repo itself.